README.md

```diff
@@ -178,17 +213,37 @@ curl --header "Content-Type: application/json" \
   http://localhost:3000/gophers
 ```
 
-### Docker
+## CORS
 
-The application is also available through [Docker](https://hub.docker.com/r/friendsofgo/killgrave), just run:
+If you want to use `killgrave` on your client application you must consider to configure correctly all about CORS, thus we offer the possibility to configure as you need through a config file.
 
-```bash
-docker run -it --rm -p 3000:3000 friendsofgo/killgrave
-```
+In the CORS section of the file you can found the next options:
 
-Remember to use the [-p](https://docs.docker.com/engine/reference/run/) flag to expose the container port where the application is listening (3000 by default).
+- **methods** (string array)
+  
+  Represent the **Access-Control-Request-Method header**, if you not specified or leave as empty array the default value will be:
```

Comment:
if you don't specify it or you leave it as any empty array, the default value will be*

Lo mismo para el resto de opciones